cmd/validate_lagoonyml.go

```diff
@@ -85,11 +85,9 @@ func init() {
 // ValidateCronjob returns an error if the command for the cronjob has any
 // newlines, and nil otherwise.
 func ValidateCronjob(c *lagoon.Cronjob) error {
-	command := strings.TrimSpace(c.Command)
-
-	if strings.Contains(command, "\n") {
+	if strings.Contains(strings.TrimSpace(c.Command), "\n") || strings.HasSuffix(c.Command, "\n") {
```

Comment:
The root cause of the error is that the validator trimmed the command string, but the templating does not. I think a more accurate test case would be:

```suggestion
	if strings.Contains(c.Command, "\n") {
```

The alternative option would be to not change the validator, and instead trim the command when templating inpod and native cronjobs. This would allow users to use the `>` yaml syntax to have a more readable cronjob definition and prevent newlines from being used.

Reply:
Are you going to also update the template component?

Reply:
@shreddedbacon

I've had a quick look.  It might be best to update the generator instead of the templates.  

https://github.com/jnsalsa/build-deploy-tool/commit/857e696d53dbf4e35cb726943459488550c95f7a#diff-65faa4119ec92040bd5c7fd2340103c542310e8e6c846c2ff676333917a2fa51L528-R531

Reply:
Thanks, just updated again.